spec/javascripts/BowlingGameSpec.js

```diff
@@ -0,0 +1,70 @@
+describe('Bowling', function() {
+  it('Gutter game', function() {
+    const bowling = new Bowling();
+
+    const times = 20;
+    for (let i=0; i < times; i++) {
+      bowling.roll(0);
+    };
+
+    expect(bowling.result()).toEqual(0);
+  });
+
+  it('adds a frame result into array after two rolls', function(){
```

Comment:
Great tests, shows you've started off small and used good TDD to build your program!